wordpress-mobile/gutenberg-mobile
# Include strings referenced in GB-mobile files in localization strings files

In the file `strings-overrides.js` ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/strings-overrides.js)) we have some strings which are not currently translated due to not being included in the localization strings files. This PR adds new `make-pot` commands to extract strings also from source files of GB-mobile repository.

**Strings referenced in GB-mobile:**
- https://github.com/wordpress-mobile/gutenberg-mobile/blob/313e99374c6204b3ec73c74981fbf7f9b152c92c/src/strings-overrides.js#L19-L21
- https://github.com/wordpress-mobile/gutenberg-mobile/blob/313e99374c6204b3ec73c74981fbf7f9b152c92c/src/strings-overrides.js#L27-L29
- https://github.com/wordpress-mobile/gutenberg-mobile/blob/313e99374c6204b3ec73c74981fbf7f9b152c92c/src/strings-overrides.js#L55

These strings are mainly used in the unsupported block editor modal:

<img src=https://user-images.githubusercontent.com/14905380/147674477-0d193775-3876-43ce-811e-754a4a3ab39c.jpeg width=350>

**To test:**
1. Run command `npm run genstrings`.
2. Observe that the localization strings files (`bundle/ios/GutenbergNativeTranslations.swift` and `bundle/android/strings.xml`) remain unmodified.
**NOTE:** Most likely the string `Text color` would be added if https://github.com/wordpress-mobile/gutenberg-mobile/pull/4409 is not already merged, please omit this string as it will be included on that PR.

PR submission checklist:
